lib/Doctrine/DBAL/Exception/MissingSQLType.php

```diff
@@ -12,7 +12,7 @@ final class MissingSQLType extends SQLParserUtilsException
     public static function new(string $typeName) : self
     {
         return new self(
-            sprintf('Value for :%1$s not found in types array. Types array key should be "%1$s"', $typeName)
+            sprintf('Value for :%1$s not found in types array. Types array key should be "%1$s"', $typeName) // what is this syntax?
```

Comment:
Please remove the comment. The second sentence looks grammatically incorrect. Not even sure what it means.

Reply:
I will look at where this exception is being thrown and take a stab at writing a better message.

Reply:
Take a look at it now and let me know what you think.